process/economics/economicsData.go

```diff
@@ -337,6 +343,13 @@ func (ed *economicsData) ComputeRelayedTxFees(tx data.TransactionWithFeeHandler)
 func (ed *economicsData) getTotalFeesRequiredForInnerTxs(innerTxs []data.TransactionHandler) *big.Int {
 	totalFees := big.NewInt(0)
 	for _, innerTx := range innerTxs {
+		if ed.isMoveBalance(innerTx) {
+			innerTxFee := ed.ComputeMoveBalanceFee(innerTx)
```

Comment:
is this function used in other places than transactionProcessor ?

Reply:
it is also called from `ComputeTxFee`